src/main/java/graphql/Directives.java

```diff
@@ -31,16 +31,25 @@
 @PublicApi
 public class Directives {
 
-    private static final String SPECIFIED_BY = "specifiedBy";
     private static final String DEPRECATED = "deprecated";
+    private static final String INCLUDE = "include";
+    private static final String SKIP = "skip";
+    private static final String SPECIFIED_BY = "specifiedBy";
     private static final String ONE_OF = "oneOf";
     private static final String DEFER = "defer";
 
-    public static final String NO_LONGER_SUPPORTED = "No longer supported";
     public static final DirectiveDefinition DEPRECATED_DIRECTIVE_DEFINITION;
+    public static final DirectiveDefinition INCLUDE_DIRECTIVE_DEFINITION;
+    public static final DirectiveDefinition SKIP_DIRECTIVE_DEFINITION;
     public static final DirectiveDefinition SPECIFIED_BY_DIRECTIVE_DEFINITION;
     @ExperimentalApi
     public static final DirectiveDefinition ONE_OF_DIRECTIVE_DEFINITION;
+    @ExperimentalApi
+    public static final DirectiveDefinition DEFER_DIRECTIVE_DEFINITION;
+
+    public static final String BOOLEAN = "Boolean";
+    public static final String STRING = "String";
+    public static final String NO_LONGER_SUPPORTED = "No longer supported";
```

Comment:
In more recent code, as a general rule we prefer to not have random strings hanging about. Some parts of this class haven't been changed in a long while and still have strings. If you prefer I can give the `public static final String` treatment to all the strings in this class